widgets/src/widgets/topStats.ts

```diff
@@ -6,20 +6,28 @@ import type { TopStatsWidgetParams } from '@/interfaces'
 
 export class TopStatsWidget extends Widget implements TopStatsWidgetParams {
   color: (typeof WIDGET_COLORS)[number]
+  withPopup?: boolean
+  version?: 'v2'
 
   constructor(params: WidgetConfig & TopStatsWidgetParams) {
     super(params)
     this.color = params.color
+    this.withPopup = params.withPopup
+    this.version = params.version
   }
 
   get topStatsWidgetRequestBody(): TopStatsWidgetParams {
```

Comment:
the naming it's a but weird here, since it's in contradiction with the return type and with the actual usage (as the version is not in the body), but i can leave with it :)